examples/api-tests/src/file-search.spec.js

```diff
@@ -60,6 +61,62 @@ describe('file-search', function () {
 
         });
 
+        describe('#filterAndRange', () => {
+
+            it('should return the default when not searching', () => {
+                const filterAndRange = quickFileOpenService['filterAndRange'];
+                assert.equal(filterAndRange, quickFileOpenService['filterAndRangeDefault']);
+            });
+
+            it('should update when searching', () => {
+                const cancelIndicator = new CancellationTokenSource();
+                const token = cancelIndicator.token;
+                quickFileOpenService['getPicks']('a:2:1', token); // perform a mock search.
```

Comment:
Can be simplified since none of the variables are re-used:

```suggestion
                quickFileOpenService['getPicks']('a:2:1',  new CancellationTokenSource().token); // perform a mock search.
```